README.rst

```diff
@@ -80,7 +80,9 @@ provided below, with descriptions of each input value field. For a concrete exam
             "sensor-res": int, Number of pixels per side of the square sensor,
             "sensor-dim": float, Length of side of the square sensor in micrometers,
             "curve": "string or null, path to a depth-dependent dye attenuation curve file. If null
-                      then `insilico_vsdi/data/RH1691-cortical-penetration-mouse.txt` is used",
+                      then `insilico_vsdi/data/RH1691-cortical-penetration-mouse.txt` is used". This
+                      file was generated by digitizing the black curve in Fig. 2L in the `paper
```

Comment:
Why is `2L` highlighted ?

Reply:
Do you know?

Reply:
It is because Github highlights invalid javascript syntax with this.